api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -243,23 +249,29 @@ message ExtAuthzPerRoute {
   }
 }
 
-// Extra settings for the check request. You can use this to provide extra context for the
-// external authorization server on specific virtual hosts \ routes. For example, adding a context
-// extension on the virtual host level can give the ext-authz server information on what virtual
-// host is used without needing to parse the host header. If CheckSettings is specified in multiple
-// per-filter-configs, they will be merged in order, and the result will be used.
+// Extra settings for the check request.
+//
+// .. note::
+//
+//   These settings are only applied to a filter configured with a
+//   :ref:`grpc_service<envoy_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.grpc_service>`.
 message CheckSettings {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.http.ext_authz.v2.CheckSettings";
 
   // Context extensions to set on the CheckRequest's
   // :ref:`AttributeContext.context_extensions<envoy_api_field_service.auth.v3.AttributeContext.context_extensions>`
   //
-  // Merge semantics for this field are such that keys from more specific configs override.
-  //
-  // .. note::
+  // You can use this to provide extra context for the external authorization server on specific
+  // virtual hosts/routes. For example, adding a context extension on the virtual host level can
+  // give the external authorization server information on what virtual host is used without needing
+  // to parse the host header. If CheckSettings is specified in multiple per-filter-configs, they
+  // will be merged in order, and the result will be used.
   //
-  //   These settings are only applied to a filter configured with a
-  //   :ref:`grpc_service<envoy_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.grpc_service>`.
+  // Merge semantics for this field are such that keys from more specific configs override.
   map<string, string> context_extensions = 1;
+
+  // Allow to override how request body is packed. If true, the body sent to the external
+  // authorization service is set with raw bytes, instead of UTF-8 string.
+  google.protobuf.BoolValue pack_as_bytes = 2;
```

Comment:
Just so I understand, this is a WKT because it can override the main option? 

Do you think this actually needs to be overriden? I find it very difficult to believe that anyone would ever override this since their impl would expect it one way or the other. Perhaps remove the override for now until someone wants it?

Reply:
Yeah. I thought I saw one comment wants this, but I couldn't find it now. 😅.